### PR TITLE
blog from julie lowndes

### PR DESCRIPTION
**Pull request by jules32**
Blog post, cross-posted at openscapes.org. 

Unfortunately I wasn't able to run the roblog checks.

### CODE REVIEW

**Comment by jules32** [comment]
@ropensci/blog-editors, it is ready for review!

File: content/blog/2020-02-21-ropensci-leadership/index.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+slug: "ropensci-leadership"
+title: rOpenSci's leadership in #rstats culture
+authors:
+  - Julie Lowndes
```
Comment (stefaniebutland) [inline content/blog/2020-02-21-ropensci-leadership/index.md]:
```suggestion
  - Julia Stewart Lowndes
```

change author name to link to author page

File: content/blog/2020-02-21-ropensci-leadership/index.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+---
+slug: "ropensci-leadership"
+title: rOpenSci's leadership in #rstats culture
```
Comment (stefaniebutland) [inline content/blog/2020-02-21-ropensci-leadership/index.md]:
```suggestion
title: rOpenSci's Leadership in #rstats Culture
```

title in title case

File: content/blog/2020-02-21-ropensci-leadership/index.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+authors:
+  - Julie Lowndes
+date: 2020-02-21
+categories: blog
+topicid: 1960
+tags:
+- R
+- community
+- unconf
```
Comment (stefaniebutland) [inline content/blog/2020-02-21-ropensci-leadership/index.md]:
```suggestion
- unconf
- unconf15
- welcome
```

File: content/blog/2020-02-21-ropensci-leadership/index.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
...at they hatched the idea for their excellent [Not So Standard Deviations podcast](http://nssdeviations.com/) following their reunion at the [2015 rOpenSci unconf, ("runconf15")](https://unconf15.ropensci.org/). That statement went straight to my heart because it pin-pointed how I had been feeling throughout the week of RStudio Conference that I had been unable to name. At rstudio::conf, I was surrounded by so many of the incredible people I had met at that very same runconf15. These folks are visionaries and leaders, founding and leading global efforts in open source software and inclusive culture, and the fact that they were all together at a small event convened by rOpenSci holds great significance. I am so honored to know this community, and to consider them allies and friends. The RStudio Conference ("rstudio::conf"), a conference with 2400 people, felt cozy with their presence and with the visible efforts they have led to make R and beyond a welcoming, innovative space. In a follow-up to an earlier [blog summary of rstudio::conf(2020)](/blog/2020/02/04/rstudioconf-2020/), here I want to reflect on how important runconf15 was, and how truly unique and gamechanging rOpenSci is."
```
Comment (stefaniebutland) [inline content/blog/2020-02-21-ropensci-leadership/index.md]: On our site the preface gets highlighted like this. Once the deploy error is fixed and we see the preview, if yellow preface is "ugly", maybe move "At their closing ...." to italics at start of post?


Screen Shot 2020-02-21 at 9 15 52 AM

File: content/blog/2020-02-21-ropensci-leadership/index.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
+---
+slug: "ropensci-leadership"
+title: rOpenSci's Leadership in #rstats Culture
```
Comment (jules32) [inline content/blog/2020-02-21-ropensci-leadership/index.md]: title: "rOpenSci's Leadership in #rstats Culture"

Comment (stefaniebutland) [inline content/blog/2020-02-21-ropensci-leadership/index.md]:
```suggestion
title: "rOpenSci's Leadership in #rstats Culture"
```

yes, title in quotes works. can commit this now

**Pull request merged**